backend/backend-cft.yaml

```diff
@@ -0,0 +1,175 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for testbed backend instance and rds connection
+Parameters:
+  HostPort:
+    Type: String
+    Default: 4500
+  ParamContainerPort:
+    Type: String
+    Default: 4500
+  ApiImage:
+    Type: String
+    Default: ga4gh/ga4gh-testbed-api:test
+  VpcId:
+    Type: String
+    Default: vpc-0d772394df78b1311
+  SubnetA:
+    Type: String
+    Default: subnet-0be7a1fb3a96191b4
+  SubnetB:
+    Type: String
+    Default: subnet-0c465ec61b98bffac
+  SubnetC:
+    Type: String
+    Default: subnet-07b0e79f4d9bbf9d0
+  DomainName:
+    Type: String
+    Default: testbed.ga4gh.org
+  DBHost:
+    Type: String
+    Default: dev-ga4gh-testbed-database-database-bqdkifegcrpe.c0q4keadjybw.us-east-2.rds.amazonaws.com
+  DBName:
+    Type: String
+    Default: testbed_db
+  DBPort:
+    Type: String
+    Default: 5432
+  DBUser:
+    Type: String
+    Default: testbed_writer
+  DBPass:
```

Comment:
Please remove this parameter as it is unused

Reply:
Since it is a password, I have moved this to System Environemnt Variables which could be provided during the execution of CF template, however I have add its usage in task definition replacing the resolve statement.

Reply:
Ah I see the confusion. DBPass is both defined as a parameter and an env variable. The parameter is not being referenced anywhere, which can be safely removed while keeping the env variable. 

Please correct me if my understanding is incorrect.

Reply:
Currently parameter is used as a place holder for environment variable, so during run time value passed as parameter will be used as ENV variable. So removing is not an option for now, it will be at a later stage when we integrate AWS SDK with Backend to have ENV variables taken care in JAVA configurations itself.

Reply:
Got it, thanks! In any case, would you be able to test this theory in practice if you did not already?

Reply:
Yes I did, So Environment variables are replaced with the values provided in parameters.